internal/worker_runner.go

```diff
@@ -19,8 +19,16 @@ type WorkerRunner interface {
 	Run()
 }
 
+// Device represents the device information that is used in the worker runner
+// to construct the device event
+type Device struct {
+	SoftwareVersion string
```

Comment:
let's set json names for it - looks like it will be serialized later?

Reply:
This is entirely internal struct, it will be remapped later to [model.Device](https://github.com/DIMO-Network/edge-network/blob/cb73571666462aa0aacf79d0baaa05e457874e26/internal/models/network.go#L37-L36)